ddspipe_core/include/ddspipe_core/types/dds/SpecificEndpointQoS.hpp

```diff
@@ -15,8 +15,8 @@
 #pragma once
 
 #include <fastdds/dds/core/policy/QosPolicies.hpp>
-#include <fastdds/rtps/common/InstanceHandle.h>
-#include <fastdds/rtps/common/Types.h>
+#include <fastdds/rtps/common/InstanceHandle.hpp>
+#include <fastdds/rtps/common/Types.hpp>
```

Comment:
I don't see why you need to include this headers.

```suggestion
#include <fastdds/rtps/common/InstanceHandle.hpp>
#include <fastdds/rtps/common/Types.hpp>
```